.github/workflows/test.yml

```diff
@@ -280,6 +280,7 @@ jobs:
             "./external-api",
             "./trade-finance-manager-api",
             "./libs/common",
+            "./azure-functions/acbs-function"
```

Comment:
this is going to change as the result of DTFS2-7014 PR -- someone is going to have to update if they merge last @abhi-markan